### PR TITLE
Fix storybook and add more components

### PR DESCRIPTION
**Pull request by lukaw3d**

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -14,4 +17,10 @@ module.exports = {
   features: {
     emotionAlias: false,
   },
+  async webpackFinal(config) {
+    config.resolve ??= {}
```
Comment (lubej) [inline .storybook/main.js:21]: Not supported in vanilla JS. Which breaks the build.

Comment (lubej) [inline .storybook/main.js:21]:
```suggestion
    config.resolve = config.resolve ?? {}
```

Comment (lukaw3d) [reply .storybook/main.js:21]: :+1: Will degrade to support older nodejs versions

File: .storybook/main.js [hunk 2]
```diff
@@ -10,8 +13,16 @@ module.exports = {
   framework: '@storybook/react',
   core: {
     builder: '@storybook/builder-webpack5',
+    disableTelemetry: true,
```
Comment (lukaw3d) [inline .storybook/main.js:16]: they had telemetry enabled by default :O

**Review by lubej — APPROVED** [review]

**Pull request merged**